helper/release_enarx.sh

```diff
@@ -450,17 +326,9 @@ if [[ "$DRYRUN" == "false" ]]; then
     git tag --sign -m "chore(release): release v${NEW_VERSION}" "v${NEW_VERSION}"
     git push upstream "v${NEW_VERSION}"
 
-    # Stage 9: Publish crates
-    shout "Please verify if package manifest looks good prior to publishing:"
-    cargo package --list
-    should_continue "true"
-    echo "Publishing"
-    publish
-    sleep 10
-
-    # Stage 10: Check if can install from released crates
+    # # Stage 10: Check if can install from released crates
```

Comment:
Is this intentional?

Reply:
Unintentional, thank you for pointing it out. Fixed.